app/views/layouts/application.html.erb

```diff
@@ -34,13 +34,17 @@
 
 
       <li class="has-dropdown">
-        <% if current_user %>
+        <% if user_signed_in? %>
           <a href="#"><%= current_user.email %></a>
           <ul class="dropdown">
             <li><%= link_to "My profile", user_path(current_user) %></li>
             <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+            <% if current_user.admin? %>
+            <li> <%= link_to "Admin Section", '#' %> </li>
+            <% end %>
           </ul>
-        <% else %>
+
+        <% elsif !user_signed_in? %>
```

Comment:
`else` is fine here, since it covers all of the same conditions as `elsif !user_signed_in?` - anytime this second condition check is hit, we KNOW that the user isn't signed in, so it's just redundant overhead to check for something we know is true